src/core/MOM_isopycnal_slopes.F90

```diff
@@ -45,6 +46,16 @@ subroutine calc_isoneutral_slopes(G, GV, US, h, e, tv, dt_kappa_smooth, &
   real, dimension(SZI_(G),SZJB_(G),SZK_(GV)+1), &
                                      optional, intent(inout) :: N2_v !< Brunt-Vaisala frequency squared at
                                                                      !! interfaces between v-points [L2 Z-2 T-2 ~> s-2]
+  real, dimension(SZIB_(G),SZJ_(G),SZK_(GV)+1), &
+                                     optional, intent(inout) :: dzu  !< Z-thickness at u-points [Z ~> m]
+  real, dimension(SZI_(G),SZJB_(G),SZK_(GV)+1), &
+                                     optional, intent(inout) :: dzv  !< Z-thickness at v-points [Z ~> m]
+  real, dimension(SZIB_(G),SZJ_(G),SZK_(GV)+1), &
+                                     optional, intent(inout) :: dzSxN !< Z-thickness times zonal slope contribution
+                                                                     !! to Eady growth rate at u-pints. [Z T-1 -> m s-1]
+  real, dimension(SZI_(G),SZJB_(G),SZK_(GV)+1), &
```

Comment:
I think that someone had been thinking about hitting the pub a bit too much when describing these variables in terms of u-pints and v-pints...

Reply:
Comments fixed in e8e200994f04ce92c93900378d7f8878a3c3339c